buildSrc/build.gradle

```diff
@@ -4,15 +4,12 @@ apply plugin: "groovy"
 repositories {
     google()
     mavenCentral()
-    gradlePluginPortal()
 }
 
 dependencies {
     implementation gradleApi()
     implementation localGroovy()
-
-    api "com.google.guava:guava:27.0.1-jre"
-    api 'org.jetbrains:annotations:16.0.2'
-    implementation "org.ow2.asm:asm-tree:9.2"
-    implementation 'com.android.tools.build:gradle:7.1.0'
+    implementation libs.gradlePlugin.android
+    implementation libs.google.guava
```

Comment:
Could you keep to use `api` instead of `implementation` for guava? At some occasion, we don't want to release Robolectric with guava.

Reply:
`implementation` is transitive instead of `api`, right?

Reply:
https://docs.gradle.org/current/userguide/java_library_plugin.html#sec:java_library_separation

> Dependencies appearing in the api configurations will be transitively exposed to consumers of the library, and as such will appear on the compile classpath of consumers. Dependencies found in the implementation configuration will, on the other hand, not be exposed to consumers, and therefore not leak into the consumers' compile classpath.

Looks like `api` is  transitively, but `implementation` not.

Reply:
Miss-spelled here, I want to say `api` is transitive instead of `implementation`.
So `api` guava before was exposed to consumers, right?

Reply:
Yup.

Reply:
This can be solved now?